data/sql/updates/pending_db_world/Crusader_pm.sql

```diff
@@ -0,0 +1,4 @@
+
+-- Remove Phase Mask 1 and add Phasemask 2.
```

Comment:
You can make this into 1 line.

UPDATE `creature` SET `phaseMask` = `phaseMask` &~1 | 2 WHERE `id1` = 28529 AND `guid` IN (128641,128697);

Removes 1 than adds 2